sandbox_externalkey_unix.go

```diff
@@ -9,19 +9,20 @@ import (
 	"io/ioutil"
 	"net"
 	"os"
+	"path/filepath"
```

Comment:
Can you run goimport ? that will keep import lines in alphabetical order.

Reply:
Isnt this already alphabetical?

Reply:
Agree.. it's fine.  goimports wouldn't group it differently and alphabetizes based on the whole string, not just the last part of the path.

Reply:
got carried away with last name filepath :)